docs/source/quickstart.rst

```diff
@@ -1,22 +1,297 @@
+.. _Quickstart:
+
 **********
 Quickstart
 **********
 
-Under Construction
+If you want to get up and running quickly with creating an ingest package
+and running it, then the quick start guide is the best place to start. It's ok
+if you do not understand what each of the steps are doing. The tutorial
+section goes through the exact same demonstration as the quickstart, but
+it includes detailed explanations for each step.
+
+Generate a new ingest package
+=============================
+
+Each study that needs to be ingested will have a single directory that contains
+all of the configuration needed by the ingest pipeline to extract, transform,
+and load the study into a target data store or service. This is known as the
+ingest package.
+
+The first thing to do after installing the ingest library CLI
+is to create a new ingest package named ``my_study``::
+
+$ kidsfirst new --dest_dir=my_study
+
+We can run it and test to make sure everything works by using the following
+command
+
+.. code-block:: text
+
+    $ kidsfirst test my_study
+
+The logs on your screen should indicate a successful test run like this:
+
+.. code-block:: text
+
+    ======================================== 1 passed in 0.12s ============================================================
+    2020-02-18 17:40:36,039 - DataIngestPipeline - Thread: MainThread - INFO - ✅ User defined data validation tests passed
+    2020-02-18 17:40:36,039 - DataIngestPipeline - Thread: MainThread - INFO - END data ingestion
+    2020-02-18 17:40:36,039 - kf_lib_data_ingest.app.cli - Thread: MainThread - INFO - ✅ Ingest pipline passed validation!
+
```

Comment:
```suggestion

So far your ingest package is pretty basic. It has 1 source data file and the Python modules needed to extract, clean and transform the data for the target service. If you inspect it, it will look like this::

    my_study/
    ├── ingest_package_config.py
    ├── transform_module.py
    ├── data
    │   └── clinical.tsv
    ├── extract_configs/
    │   └── extract_config.py
    └── tests
        ├── conftest.py
        └── test_custom_counts.py
```